test_runner/src/main/kotlin/ftl/args/ValidateIosArgs.kt

```diff
@@ -17,6 +18,10 @@ private fun IosArgs.assertMaxTestShards() { this.maxTestShards
         "max-test-shards must be >= ${IArgs.AVAILABLE_PHYSICAL_SHARD_COUNT_RANGE.first} and <= ${IArgs.AVAILABLE_PHYSICAL_SHARD_COUNT_RANGE.last}, or -1. But current is $maxTestShards"
     )
 }
+private fun IosArgs.assertTestTypes() {
+    if (xctestrunFile.isNullOrBlank() or xctestrunZip.isNullOrBlank())
```

Comment:
`isBlank()` should be enough in both cases

Reply:
In this file, we have two logic operator convention now. Look:

```
private fun AndroidArgs.assertShards() {
    if (numUniformShards != null && maxTestShards > 1) throw FlankConfigurationError(
        "Option num-uniform-shards cannot be specified along with max-test-shards. Use only one of them."
    )
}

private fun AndroidArgs.assertTestTypes() {
    if (!(isRoboTest or isInstrumentationTest)) throw FlankConfigurationError(
        "One of following options must be specified [test, robo-directives, robo-script]."
    )
}

private fun AndroidArgs.assertRoboTest() {
    // Using both roboDirectives and roboScript may hang test execution on FTL
    if (roboDirectives.isNotEmpty() && roboScript != null) throw FlankConfigurationError(
        "Options robo-directives and robo-script are mutually exclusive, use only one of them."
    )
}

```

Maybe we should decide and keep only one?

Reply:
👍 @adamfilipow92 we can consider it in another issue. This PR name is `Improve error message on IOS` not `Android`

Reply:
Right, my mistake. I update my comment to iOS version.

Reply:
Shall i create a ticket for this @jan-gogo @adamfilipowicz92

Reply:
@Sloox hehe, ofc not if it is only about style convention ;). It is minor for me, you can leave it as-is if you find it readable or fix if you want.